generators/client/templates/angularjs/src/main/webapp/app/components/util/_data-util.service.js

```diff
@@ -77,7 +77,13 @@
         }
 
         function openFile (type, data) {
-            $window.open('data:' + type + ';base64,' + data, '_blank', 'height=300,width=400');
+            var fileURL = 'data:' + contentType +';base64,' + data;
+            var win = window.open();
```

Comment:
you should use `$window.open('','_blank')`

Reply:
this should then be fixed here as well?  https://github.com/jhipster/ng-jhipster/blob/master/src/service/data-util.service.ts

Reply:
There is an error in the fix.

The name of the first parameter of the function is "type" but on line 80 it is used as "contentType".

Apart from this, the fix works !

Thanks a lot for your work!

Reply:
@stefanobertini i know thats already corrected
anyway here  https://github.com/jhipster/ng-jhipster/blob/master/src/service/data-util.service.ts
@sendilkumarn s open comman is not used maybe update that although i am not sure if that is necessary

Reply:
aah good catch. This should be updated

Reply:
@sendilkumarn can you first check the behavior without `_blank` I have a feeling it works the same since we use window.open

Reply:
check here https://developer.mozilla.org/en-US/docs/Web/API/Window/open

Reply:
window.open it will work. But with _blank is preferred.